src/app/home/pages/PlansPage/components/Wizard/NamespacesTable.tsx

```diff
@@ -97,9 +97,25 @@ const NamespacesTable: React.FunctionComponent = () => {
     const editedNamespace = values.editedNamespaces.find(
       (editedNS) => editedNS.id === namespace.id
     );
+    const { currentPlan } = useSelector((state: DefaultRootState) => state.plan);
+    let isTargetNamespaceNameSet = false
+    let setTargetNamespaceName = ""
+    if (currentPlan != null) {
```

Comment:
Try to avoid using `!=` or `==` in JS/TS, they perform type coercion that is usually undesirable. We should use `!==` or `===` instead.

In this case though, you can just do `if (currentPlan)` because a non-null plan is truthy.